retrieval_service/datastore/providers/cloudsql_postgres.py

```diff
@@ -330,17 +330,34 @@ async def get_amenity(self, id: int) -> Optional[models.Amenity]:
         return res
 
     async def amenities_search(
-        self, query_embedding: list[float], similarity_threshold: float, top_k: int
+        self,
+        query_embedding: list[float],
+        similarity_threshold: float,
+        top_k: int,
+        filter_time: Optional[str],
+        filter_day: Optional[str],
     ) -> list[models.Amenity]:
+
+        filter_time_datetime = None
+        filter_query = "WHERE "
+        if filter_time and filter_day:
```

Comment:
What happens if this is one but not the other? will it just be ignored?

Reply:
It's checked in routes.py. Will throw an error if one is given but not the other.